engine/session_live.js

```diff
@@ -8,6 +8,14 @@ const { m3u8Header } = require("./util.js");
 const { AbortController } = require("abort-controller");
 
 const timer = (ms) => new Promise((res) => setTimeout(res, ms));
+const daterangeAttribute = (key, attr) => {
+  if (key === "planned-duration" || key === "duration") {
+    console.log("Attribute is: " + attr + "Parsed gives us: " + parseFloat(attr));
```

Comment:
Can we remove this log print before we merge?

Reply:
Missed that, of course it is now removed.